package/src/components/Message/MessageSimple/MessageSimple.tsx

```diff
@@ -234,16 +262,53 @@ const MessageSimpleWithContext = <
             {message.pinned ? <MessagePinnedHeader /> : null}
           </View>
 
-          <View style={[styles.contentWrapper, contentWrapper]}>
-            <MessageContent
-              backgroundColor={backgroundColor}
-              noBorder={noBorder}
-              setMessageContentWidth={setMessageContentWidth}
-            />
-            {reactionListPosition === 'top' && ReactionListTop ? (
-              <ReactionListTop messageContentWidth={messageContentWidth} />
-            ) : null}
-          </View>
+          {alignment === 'left' ? (
```

Comment:
Cool ! So now both orientations should be supported, thanks for this ! 

However, from what I can see this still doesn't get rid of `Swipeable` if we have `enableSwipeToReply` set to `false`, right ?

Reply:
Yep, let me see if I can fix that case and remove Swipeable from the hierarchy

Reply:
I have pushed it @isekovanic